index.html

```diff
@@ -2071,6 +2071,7 @@ <h2>Extensions</h2>
 </p>
 </section>
 </section>
+</section>
```

Comment:
I went through this carefully and I think this should be on line 2030.
The existing spec already has section 4.8 Error Handling.
The new APNG information was added just before this section. But it wasn't closed completely. As a result, the Error Handling section is currently 5.9.4 (3 levels deep). It should only be 2 levels deep.

If this closing section tag were on line 2030, it would close out the APNG information and keep the Error Handling section 2 levels deep (5.9 Error Handling).